ARCHITECTURE.md

```diff
@@ -64,12 +64,18 @@ The `localStorage` state is updated automatically on every Redux store update, v
 
 ![Kedro-Viz data flow diagram](/.github/img/app-architecture-data-flow.png)
 
-On initialisation, Kedro-Viz [manually normalises pipeline data](/src/store/normalize-data.js), in order to [make immutable state updates as performant as possible](https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape).
+Kedro-viz currently utilizes two different methods of data ingestion: the Redux setup for ingesting data for the pipeline and flowchart-view related components, and Graphql via Apollo Client for ingesting data for the experiment tracking components. 
+
+On initialisation, for the redux setup, Kedro-Viz [manually normalises pipeline data](/src/store/normalize-data.js), in order to [make immutable state updates as performant as possible](https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape).
 
 Next, it [initialises the Redux data store](https://github.com/quantumblacklabs/kedro-viz/blob/main/src/store/initial-state.js), by merging this normalised state with other data sources such as saved user preferences from `localStorage`, URL flags, and default values.
 
 During preparation, the initial state is separated into two parts: pipeline and non-pipeline state. This is because the non-pipeline state should persist for the session duration, even if the pipeline state is reset/overwritten - i.e. if the user selects a new top-level pipeline.
 
+All experiment tracking data are stored in a SQLite database that is generated automatically once [experiment tracking is enabled in your Kedro project](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html). On loading Kedro-viz from the Kedro project, the Kedro-viz backend will consume the run data stored in the database and serve the data via the graphql endpoint via graphql query requests from the Apollo client on the Front end. 
```

Comment:
Hi, I just realised how to enable experiment tracking in your Kedro project is not covered in the docs. It is covered in our Kedro-Viz Readme. 

Do we update that part of the Kedro docs? @yetudada

Reply:
Heya Rashida, I think [this section](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html#enable-experiment-tracking) on the kedro docs will cover the setup to enable experiment tracking in the Kedro docs. ( it will need some more updating to be tied in with recent development with Kedro-viz, which which will be part of the PR that I will be putting out in my experiment tracking tutorial.) The tutorial for experiment tracking that I am setting up will also cover this as well.